src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/SymbolAnalyzerBase.cs

```diff
@@ -16,7 +16,7 @@ public abstract class SymbolAnalyzerBase : DiagnosticAnalyzer
 
         protected INamedTypeSymbol ClientOptionsType { get; private set; }
 
-        public override void Initialize(AnalysisContext context)
+        public sealed override void Initialize(AnalysisContext context)
```

Comment:
Curious: why make this sealed?

Reply:
It was `sealed` before. I removed it in my previous PR to do some change. Now this change is revert so the `sealed` need to be added back.